composer.json

```
@@ -3,7 +3,7 @@
     "type": "neos-package",
     "name": "psmb/splitadd",
     "require": {
-        "neos/neos": "^8.0 || ^7.0 || ^5.0 || ^4.0 || ^3.3"
+        "neos/neos": "^8.2"
```

we would need to test this but the plugin _should_ work with all neos ui versions.

The version number of the extensibility stuff is currently always backwardscompatible - only not always forwards though. So techically even a Neos 5 should work but imo we shouldnt put any effort in supporting this legacy system :P

```suggestion
        "neos/neos-ui": "~7.3 || ^8.0"
```

---

But yes good point

> The ckeditor-exports import might have been breaking at some point...

indeed they did at some time, but the  ckeditor-exports  are available at least since 7.3
A thing that potentially can cause trouble is that neos < 8.2 provides a ckeditor 12 while neos 8.2 and 8.3 ship ck16. But if we find a incompatibility while testing i would just suggest to drop support for prior versions, as neos < 8.3 will be out of date soon.